mozilla-release/browser/base/content/browser.xul

```diff
@@ -97,9 +97,9 @@
           />
 #endif
       <menuseparator/>
+      <menuitem id="context_toggleMuteTab" oncommand="TabContextMenu.contextTab.toggleMuteAudio();"/>
```

Comment:
That line was moving from line 102 to 100 because the "duplicate Tab" option should be placed right above pin tab